gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14-bin.zip
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/50f493ab1421e4fbba96bc4ef10078e5d9a0205d/gradle/wrapper/gradle-wrapper.properties#L3"></span>A newer version of Gradle than 8.14 is available: 8.14.1</td>
    </tr>
  </tbody>
</table>